changelogs/current.yaml

```diff
@@ -351,6 +351,13 @@ new_features:
   change: |
     added :ref:`include_tls_session <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.include_tls_session>`
     to support sending TLS SNI data as part of CheckRequest for authorization check.
+- area: router
+  change: |
+    add :ref:`flush upstream log on new request
+    <envoy_v3_api_field_extensions.filters.http.router.v3.Router.UpstreamAccessLogOptions.flush_upstream_log_on_new_request>`
+    to allow recording an upstream access log entry when a new HTTP request is received, and an upstream connection has been
```

Comment:
nit: upstream stream is created?

for HTTP/1 these are one and the same but for H/2 we may have a preestablished connection, and so I want to be clear :-)

Reply:
done